08.ls_object/lib/ls/formatter.rb

```diff
@@ -62,18 +61,18 @@ def build_data(file)
       }
     end
 
-    def format_long(files, file_data)
-      total_blocks = "total #{files.map(&:blocks).sum}"
-      max_lengths = %i[nlink user group size].to_h { |key| [key, find_max_length(file_data, key)] }
-      rows = file_data.map { |data| format_row(data, max_lengths) }
+    def format_long(file_data_list)
+      total_blocks = "total #{file_data_list.map { |data| data[:blocks] }.sum}"
```

Comment:
```suggestion
      total_blocks = "total #{file_data_list.sum { |data| data[:blocks] }}"
```

って書くことができます。ただ、`{`や`}`がごちゃごちゃして読みにくいのでいったん変数に入れた方がベターかも。あと、`total_blocks`だと配列っぽく見えるので、文字列とわかる変数名の方がなおベターかも。

```ruby
      total = file_data_list.sum { |data| data[:blocks] }
      total_text = "total #{total}"
```